### PR TITLE
Translation of comments (background.js + inject.js)

### PR DESCRIPTION
**Pull request by FJduFou**
ToC=Translation of comments
----------------
A lot of people don't speak / understand German (and me the first one) which is problematic when trying to understand the code in order to be able to contribute to it, I offer here the translation of the comments (via G.translate, sorry)...
_Don't hesitate to take a quick look if I haven't made a mistake, I really don't understand a word of German (except transparent words)_

### CODE REVIEW

**Comment by JhonnyBn** [comment]
This is very helpful. I find it difficult to understand the code as well

**Comment by FJduFou** [comment]
Thx, I will gradually translate everything, starting with the most important (which is relative but good ..)

File: js/inject.js [hunk 1]
```diff
@@ -60,20 +60,20 @@
...
+	// is there a translation?
 	if (Languages.PossibleLanguages[lng] === undefined) {
 		lng = 'en';
 	}
 
 	if (uLng !== null){
 		lng = uLng;
 	} else {
-		// damit man die sprache auslesen kann ohne diese über die einstellungen einmal ändern zu müssen
+		// so that the language can be read out without having to change it once via the settings
```
Comment (Th3C0D3R) [inline js/inject.js:76]: change to:
to read the language without the condition of changing it once in the settings

**Review by Th3C0D3R — COMMENTED** [review]
one line to change

**Comment by Th3C0D3R** [comment]
ok nvm xD

**Pull request merged**